hydra-node/test/Hydra/Chain/Direct/TxSpec.hs

```diff
@@ -179,6 +179,7 @@ spec =
                                           ErrNotEnoughFunds{} -> "Not enough funds"
                                           ErrUnknownInput{} -> "Unknown input"
                                           ErrScriptExecutionFailed{} -> "Script(s) execution failed"
+                                          ErrTranslationError{} -> "Transaction context translation error"
```

Comment:
Formatting loons off.. are you using fourmolu already?

Reply:
I configured it and I can see it in action. I'll show you in the meeting but for this line I don't see what is off?

Reply:
Nevermind, it was only looking weird on mobile.